src/views/tweet.nim

```diff
@@ -100,7 +100,7 @@ proc renderVideo*(video: Video; prefs: Prefs; path: string): VNode =
           renderVideoDisabled(video, path)
         else:
           let vid = video.variants.filterIt(it.contentType == video.playbackType)
```

Comment:
```suggestion
          let playbackType = if prefs.proxyVideos: video.playbackType else: mp4
          let vid = video.variants.filterIt(it.contentType == video.playbackType)
```

Reply:
This could work, my concern here would be in the odd case where mp4 isn't available, which I think is mostly the case for "pro" videos (ads and ad-like business posts) and maybe some very old videos. A fallback would be necessary, so this bit of code should be able to handle cases where mp4 isn't in the list of variants.